cli_reference/cli_manager/cli-manager-using.adoc

```diff
@@ -0,0 +1,21 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cli-manager-install_{context}"]
```

Comment:
Not related to the build failures, but I might suggest changing this module ID so that it's not so close to the `cli-manager-install` ID in the `cli-manager-install.adoc` assembly file:
```suggestion
[id="using-cli-manager_{context}"]
```

Reply:
Oops sorry, I mispoke (this is an assembly also). But still, should drop the `_{context}` from any assembly IDs (and try to match the actual title as close as possible).